docs/rules/newline-per-chained-call.md

```diff
@@ -23,13 +23,28 @@ d3
         42
     ])
     .enter()
-    .append('p')
+    .append("p")
     .text(function (d) {
         return "I'm number " + d + "!";
     });
 ```
 
-This rule reports such code and encourages new lines after each call in the chain as a good practice.
+Another argument in favor of this style is that it improves the clarity of diffs when something in the method chain is changed:
+
+```diff
+// Less readable
```

Comment:
Also, maybe pull "Less clear" out of the code block? Pretty sure diffs don't support JS-style comments.